mlflow/pyfunc/__init__.py

```diff
@@ -1853,7 +1848,7 @@ def predict(model_input: List[str]) -> List[str]:
         )
 
 
-@format_docstring(LOG_MODEL_PARAM_DOCS.format(package_name="scikit-learn"))
+@format_docstring(LOG_MODEL_PARAM_DOCS.format(package_name="pyfunc"))
```

Comment:
I think `scikit-learn` is used on purpose because there is no package named pyfunc.